recipes/libe57format/all/conanfile.py

```diff
@@ -0,0 +1,72 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+class LibE57FormatConan(ConanFile):
+    name = "libe57format"
+    license = "BSL-1.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/asmaloney/libE57Format"
+    description = "Library for reading & writing the E57 file format"
+    topics = ("e57", "io", "point-cloud")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
```

Comment:
fPIC is not honored, need a patch to remove it from upstream's cmakefile:
https://github.com/asmaloney/libE57Format/blob/v2.2.0/CMakeLists.txt#L107

Reply:
Thanks, wrote a patch for it.